src/main/java/seedu/address/logic/parser/AddAllergyCommandParser.java

```diff
@@ -20,37 +21,36 @@
 public class AddAllergyCommandParser implements Parser<AddAllergyCommand> {
     private static final Logger logger = LogsCenter.getLogger(AddCommandParser.class);
     /**
-     * Parses the given {@code String} of arguments in the context of the AddAllergyCommand
-     * and returns an AddAllergyCommand object for execution.
+     * Parses the given arguments string and creates an {@link AddAllergyCommand} object.
      *
-     * @throws ParseException if the user input does not conform the expected format.
+     * @param args the arguments string containing user input.
+     * @return An {@link AddAllergyCommand} object containing the parsed NRIC and set of allergies.
```

Comment:
In the codebase we use `{@code}` instead of `{@link}`, I am not sure if it violates any coding standards